docs/src/pages/guides/composition/composition.md

```diff
@@ -127,40 +127,70 @@ Here is a demo with [React Router DOM](https://github.com/ReactTraining/react-ro
 
 You can find the details in the [TypeScript guide](/guides/typescript#usage-of-component-property).
 
-### Caveat with refs
+## Caveat with refs
 
-Some components such as `ButtonBase` (and therefore `Button`) require access to the 
-underlying DOM node. This was previously done with `ReactDOM.findDOMNode(this)`.
- However `findDOMNode` was deprecated (which disqualifies its usage in React's concurrent mode)
-in favour of component refs and ref forwarding. 
+This section covers caveats when using a custom component as `children` or for the 
+`component` prop.
 
-It is therefore necessary that the component you pass to the `component` prop
-can hold a ref. This includes:
+Some of the components need access to the DOM node. This was previously possible
+by using `ReactDOM.findDOMNode`. This function is deprecated in favor of `ref` and
+ref forwarding. However, only the following component types can be given a `ref`:
 
-- class components
-- ref forwarding components (`React.forwardRef`)
-- built-in components e.g. `div` or `a`
+* Any Material-UI component
```

Comment:
```suggestion
- Any Material-UI component
```

Otherwise, we can accept entropy knowing we will normalize it with Prettier in the future?

Reply:
I don't think prettier cares about this. [`-`, `+` or `*` are all valid enumeration symbols](https://github.github.com/gfm/#list-items).

Reply:
Prettier replaces them all with `*`.